sale_automatic_workflow/models/sale_workflow_process.py

```diff
@@ -35,36 +35,41 @@ def _default_filter(self, xmlid):
         default='direct',
     )
     validate_order = fields.Boolean(string='Validate Order')
-    order_filter_domain = fields.Char(
+    order_filter_domain = fields.Text(
         string='Order Filter Domain',
-        default="[('state', '=', 'draft')]"
+        related='order_filter_id.domain',
```

Comment:
I'm not following - these are definitely editable as it is, and adding readonly would restrict usage IMO

Currently:
![image](https://cloud.githubusercontent.com/assets/7397578/26001223/8a9e94c4-36e0-11e7-938a-73b421847508.png)

With Readonly:
![image](https://cloud.githubusercontent.com/assets/7397578/26001230/9294d530-36e0-11e7-9339-cf91baeac669.png)